WordPress/src/main/java/org/wordpress/android/ui/stats/StatsCommentsFragment.java

```diff
@@ -82,7 +82,7 @@ protected void updateUI() {
             return;
         }
 
-        if (mDatamodels[1] != null) { // check if comment-followers is already here
+        if (mDatamodels[1] != null && !isErrorResponse(1)) { // check if comment-followers is already here
```

Comment:
Each fragment receive a notification and then update the UI when receive a datamodel Obj that it can handle.
https://github.com/wordpress-mobile/WordPress-Android/blob/issue/2219-ClassCastException-StatsCommentsFragment/WordPress/src/main/java/org/wordpress/android/ui/stats/StatsAbstractListFragment.java#L388